templates/cache-dns.template.yml

```diff
@@ -15,5 +15,6 @@
 
 hooks:
   after_bundle_exec:
+    - exec: "gem install pg redis"
```

Comment:
Ain't those gems already installed because they are listed in the Discourse Gemfile? And it's not like we will move away from pg/redis.

Reply:
I thought that would be the case, but they aren't available as the user that runs this script (`root`). They also aren't listed via `gem list`. Probably installed as a different user? 

In any case, best scenario is this command is a no-op, worst case is that gems are installed.

Reply:
Discourse app gems are bundled locally, see:

https://github.com/discourse/discourse_docker/blob/2a9faf7e5680b9979585be613162c45a0960a525/image/base/release.Dockerfile#L7-L8

So this will install those two again, in a different place

Reply:
Even though it's shipped with Discourse core, the `cache_critical_dns` script won't use any of the vendored libraries. It uses the ["system" ruby](https://github.com/discourse/discourse/blob/650adbe423ca2ed2394dd06054d50052a91e9d08/script/cache_critical_dns#L1) (and gems) in the container.

Reply:
@CvX @xfalcox would you mind either approving or suggesting a change, given the explanation for installing the gems?